config/default.yml

```diff
@@ -765,6 +765,7 @@ Rails/PluralizationGrammar:
   Description: 'Checks for incorrect grammar when using methods like `3.day.ago`.'
   Enabled: true
   VersionAdded: '0.35'
+  VersionChanged: '<<next>>'
```

Comment:
This is not a fundamental configuration change, but rather a fix for a false negative. So, this change is not necessary.

```suggestion
```

Reply:
Changed the changelog type to fix as well